roles/setup/vars/main.yml

```diff
@@ -5,48 +5,50 @@ general:
 
 # homebrew applications
 applications:
-  - autojump
+#   - autojump
   - git
   - tree
   - httpie
   - curl
   - wget
-  - kubectl
-  - kubernetes-helm
+  - postgresql
+  - rabbitmq
+#   - kubectl
+#   - kubernetes-helm
   - zsh
-  - go
+#   - go
   - vim
-  - hugo
-  - saltstack
-  - neovim
-  - ag 
-  - ctags
+#   - hugo
+#   - saltstack
+#   - neovim
+#   - ag 
+#   - ctags
   - terraform
 
 # homebrew cask applications
 cask_applications:
   - google-chrome
   - slack
   - iterm2
-  - caffeine
+#   - caffeine
   - docker
-  - minikube
+#   - minikube
   - vagrant
-  - emacs
-  - garmin-express
+#   - emacs
+#   - garmin-express
   - 1password
   - alfred
```

Comment:
This is probably controversial. (Alfred's fine but I don't use it, for example.)